cli/src/main/java/org/jboss/as/cli/CommandCompleter.java

```diff
@@ -65,7 +65,8 @@ public int complete(CommandContext ctx, String buffer, int cursor, List<String>
         if(result < 0) {
             return result;
         }
-        return result - offset;
```

Comment:
@xstefank, this would work but I am wandering if you could isolate the fix to the failing piece (value completion). If you look at the OperationRequestCompleter, line 300: https://github.com/wildfly/wildfly-core/blob/master/cli/src/main/java/org/jboss/as/cli/operation/OperationRequestCompleter.java#L300
The chunk that has been completed doesn't contain the ending whitespaces. I think that it is the issue root cause. 
Computing the length diff between the original line (the substring of the buffer starting at result index) and the chunk should capture this case.